myip.go

```diff
@@ -49,3 +50,24 @@ func GetIP() (net.IP, error) {
 
 	return nil, fmt.Errorf("Unable to find IP")
 }
+
+// GetIPUsingIfConfig_Me uses the ifconfig.me web service to get your public IP. It returns your public IP or an error.
+func GetIPUsingIfConfig_Me() (net.IP, error) {
```

Comment:
Consider removing the underscore in the function name. That is generally to be avoided unless you are using all caps or using autogenerated code like protos.

Reply:
Done.